sdk/cosmosdb/cosmos/samples-dev/ItemManagement.ts

```diff
@@ -11,29 +11,27 @@ dotenv.config({ path: path.resolve(__dirname, "../sample.env") });
 
 import { logSampleHeader, handleError, finish, logStep } from "./Shared/handleError";
 import { readFileSync } from "fs";
-import { CosmosClient } from "../dist-esm";
-const {
-  COSMOS_DATABASE: databaseId,
-  COSMOS_CONTAINER: containerId,
-  COSMOS_ENDPOINT: endpoint,
-  COSMOS_KEY: key
-} = process.env;
+import { CosmosClient } from "@azure/cosmos";
+const key = process.env.COSMOS_KEY || "<cosmos key>";
+const endpoint = process.env.COSMOS_ENDPOINT || "<cosmos endpoint>";
+const containerId = process.env.COSMOS_CONTAINER || "<cosmos container>";
+const dbId = process.env.COSMOS_DATABASE || "<cosmos database>";
 
 logSampleHeader("Item Management");
 
 const itemDefs = JSON.parse(
-  readFileSync(path.resolve(__dirname, "./Shared/Data/Families.json"), "utf8")
+  readFileSync(path.resolve(__dirname, "../assets/Data/Families.json"), "utf8")
```

Comment:
`__dirname` will be different relative to the assets folder in the JS vs TS samples. In the JS copies they are at the same level, so I would just use a relative path and assume the sample is run from the folder where assets is located.

```suggestion
  readFileSync("../assets/Data/Families.json", "utf8")
```

Reply:
I followed the first approach without the directory name

Reply:
I ended up changing this actually in my patch. It provides a nicer TypeScript experience. Hope that's okay.